### PR TITLE
Completed css-pseudo-classes

### PR DESCRIPTION
**Pull request by K-Bowler**
css-pseudo-classes

### CODE REVIEW

File: css-pseudo-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.hover {
+  padding: 15px 35px;
+  font-size: 24px;
+}
+
+.hover:hover {
+  color: white;
+  background-color: black;
+}
```
Comment (TimelikeClosure) [inline css-pseudo-classes/styles.css]: The cursor should change to a pointer when hovering over the button. Also, the button's spacing should be consistent with the example.

---

image

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: css-pseudo-classes/styles.css (hunk 1 above)
Comment (K-Bowler) [reply css-pseudo-classes/styles.css]: css-pseudo-classes-2

Comment (TimelikeClosure) [reply css-pseudo-classes/styles.css]: @K-Bowler What about the spacing?

**Comment by K-Bowler** [comment]
image

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**